tests/prefix_search_settings.test.ts

```diff
@@ -0,0 +1,176 @@
+import { afterAll, expect, test, describe, beforeEach } from "vitest";
+import { ErrorStatusCode } from "../src/types";
+import {
+  clearAllIndexes,
+  config,
+  BAD_HOST,
+  MeiliSearch,
+  getClient,
+  dataset,
+} from "./utils/meilisearch-test-utils";
+
+const index = {
+  uid: "movies_test",
+};
+
+afterAll(() => {
+  return clearAllIndexes(config);
+});
+
+describe.each([{ permission: "Master" }, { permission: "Admin" }])(
+  "Test on prefix search settings",
+  ({ permission }) => {
+    beforeEach(async () => {
+      await clearAllIndexes(config);
+      const client = await getClient("Master");
+      const { taskUid } = await client.index(index.uid).addDocuments(dataset);
+      await client.waitForTask(taskUid);
+    });
+
+    test(`${permission} key: Get prefixSearch settings on empty index`, async () => {
+      const client = await getClient(permission);
+
+      const response = await client.index(index.uid).getPrefixSearch();
+      expect(response).toMatchSnapshot();
+    });
+
+    test(`${permission} key: Set prefixSearch settings with dedicated endpoint on empty index`, async () => {
+      const client = await getClient(permission);
+
+      const { taskUid } = await client
+        .index(index.uid)
+        .updatePrefixSearch("disabled");
+      await client.index(index.uid).waitForTask(taskUid);
+
+      const updatedSettings = await client.index(index.uid).getPrefixSearch();
+      expect(updatedSettings).toBe("disabled");
```

Comment:
Same question here, why not expect to match snapshot?

Reply:
Same as before